### ISSUE TITLE
Split the "Achievement/Special" filter into two separate ones for all class page

### ISSUE THREAD

**Issue by Gonzom** [Js41637/Overwatch-Item-Tracker#221]
Could the "Achievement/Special" filter be separated into two different filters - a "Special" and an "Achievement" filter? It got to the point where it should be split as it has two groups of distinct items which would make looking for a specific item much easier.

**Comment by Js41637** [comment]
Yeah, this is just an issue in how I have things setup. All achievements, blizzard items, origins skins, etc are classified in-game as `System Unlocks` and there is no real distinction between them when I extract the data, as such, there is a lot of manual mapping done and some overriding. It's all still classified as an `achivement` though. I just kinda cheat. Origins/Blizzcon skins i set to `achivement: "blizzard"`. I had to had a new one for the mercy breast cancer ones as they are treated like an event but they are not which is `achivement: "bcf"` and everything else is just `achievement: true`. Technically it is super easy to distinguish them. I just really need to think of a better system of classifying items. At the moment there is like normal items, event items, item groups (diablo, competitive) and achievement/special items.

**Comment by Gonzom** [comment]
I'm not familiar with web coding that much (or with your code), but if you could add an enum to the data for types of items, will that help?

**Comment by Js41637** [comment]
I looked into adding this but tbh I don't think it's worth it, most heroes only have 2 achievments so making a whole filter for that feels overkill and no all heroes even have a special and if they do, it's usually only 1.

**Comment by Gonzom** [comment]
This was actually opened with All Heroes in mind as there are 15 or so special items and the rest are comp ones.

**Comment by Js41637** [comment]
Oh right, dammit.
